tests/checkpointing/test_legacy_checkpoints.py

```diff
@@ -24,9 +24,27 @@
 CHECKPOINT_EXTENSION = ".ckpt"
 
 
-# todo: add more legacy checkpoints :]
+# todo: add more legacy checkpoints :] for < v0.8
 @pytest.mark.parametrize("pl_version", [
-    "0.10.0", "1.0.0", "1.0.1", "1.0.2", "1.0.3", "1.0.4", "1.0.5", "1.0.6", "1.0.7", "1.0.8"
+    # "0.8.1",
+    "0.8.3",
+    "0.8.4",
+    "0.8.5",
+    "0.9.0",
+    "0.10.0",
+    "1.0.0",
+    "1.0.1",
+    "1.0.2",
+    "1.0.3",
+    "1.0.4",
+    "1.0.5",
+    "1.0.6",
+    "1.0.7",
+    "1.0.8",
+    "1.1.0",
+    "1.1.1",
+    "1.1.2",
+    "1.1.3",
```

Comment:
I assume the checkpoint files are already added?

Reply:
yes, I had to add the old manually, otherwise, tests won't pass ;]